exchanges/okx/okx_type_convert.go

```diff
@@ -160,6 +160,22 @@ func (a *OpenInterest) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+// UnmarshalJSON deserializes JSON, and timestamp information.
+func (a *FundingRateResponse) UnmarshalJSON(data []byte) error {
+	type Alias FundingRateResponse
+	chil := &struct {
+		*Alias
+		FundingRate string `json:"fundingRate"`
+	}{
+		Alias: (*Alias)(a),
+	}
+	err := json.Unmarshal(data, chil)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
```

Comment:
Rm as this is purging the rates when using test `TestGetFundingRates` also please add field `RealisedRate` in `FundingRateResponse`

```
{"fundingRate":"0.0001905490218961","fundingTime":"1687536000000","instId":"BTC-USD-SWAP","instType":"SWAP","realizedRate":"0.0001905452237828"}
```

Reply:
I don't know why its showing up for you that I introduced that, but its a great pickup! I would have been [assigning the incorrect historical rate](https://www.okx.com/docs-v5/en/#public-data-rest-api-get-funding-rate-history) 🦅 🎉